src/fullscreen/fullscreen-controller.js

```diff
@@ -0,0 +1,246 @@
+//@flow
+import EventManager from '../event/event-manager';
+import Player from '../player';
+import FakeEvent from '../event/fake-event';
+import * as Utils from '../utils/util';
+
+/**
+ * The IOS fullscreen class name.
+ * @type {string}
+ * @const
+ */
+const IN_BROWSER_FULLSCREEN_FOR_IOS: string = 'in-browser-fullscreen-mode';
+
+/**
+ * @class FullscreenController
+ * @param {Player} player - The player.
+ */
+class FullscreenController {
+  _player: Player;
+  _inBrowserFullscreen: boolean;
+  _isFullscreenManually: boolean;
```

Comment:
change to `_isInBrowserFullscreen` for naming consistency

Reply:
@Yuvalke ☝️

Reply:
It changed